plugins/MIDIPressureToCC/DistrhoPluginInfo.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * MIDI PBToCC plugin based on DISTRHO Plugin Framework (DPF)
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * Copyright (C) 2019 Christopher Arndt <info@chrisarndt.de>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to
+ * deal in the Software without restriction, including without limitation the
+ * rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+ * sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+ * IN THE SOFTWARE.
+ */
+
+#ifndef DISTRHO_PLUGIN_INFO_H
+#define DISTRHO_PLUGIN_INFO_H
+
+#define DISTRHO_PLUGIN_BRAND        "chrisarndt.de"
+#define DISTRHO_PLUGIN_NAME         "MIDI Channel Pressure (Aftertouch) to CC"
```

Comment:
That's a bit too long (This is gonna show up in the Hosts's plugin list).

Recommendation: "MIDI Pressure to CC"

Unfortunately, DPF doesn't support plugin descriptions via `rdfs:comment` [yet](https://github.com/DISTRHO/DPF/issues/230), but it's there in the `getDescription` method, if and when it does.

For now, let's just put the additional keywords in the description in the README.